roles/postgresql/tasks/postgresql_yum7.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: yum | Initialize postgresql
```

Comment:
Probably you meant `Install postgresql`, because next block is `Initialize postgresql`